posts/2025/farmOS-3.4.0/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: farmOS 3.4.0
+date: 2025-02-24
+author: Michael Stenta
+slug: 2025/farmOS-3.4.0
+categories:
+  - newsletter
+---
+
+# farmOS 3.4.0
+
+We are excited to announce the release of
+[farmOS 3.4.0](https://github.com/farmOS/farmOS/releases/tag/3.4.0)!
+This release includes a number of new features and improvements. Here are some
+of the highlights.
+
+**Note: farmOS 3.4.1 was released shortly after 3.4.0 with some Drupal core
+security fixes. If you are upgrading from an earlier version of farmOS, please
+use [3.4.1](https://github.com/farmOS/farmOS/releases/tag/3.4.1) instead.**
```

Comment:
This is a funny predicated statement - sort of makes it seem like there's some scenario where you wouldn't use 3.4.1.

Maybe something like:

```suggestion
security fixes. Please
use [3.4.1](https://github.com/farmOS/farmOS/releases/tag/3.4.1) in
preference to 3.4.0 when installing or upgrading.**
```